internal/webserver/handler-api.go

```diff
@@ -225,6 +225,14 @@ func (h *Handler) ApiInsertBookmark(w http.ResponseWriter, r *http.Request, ps h
 	// Save bookmark to database
 	results, err := h.DB.SaveBookmarks(ctx, true, *book)
 	if err != nil || len(results) == 0 {
+		if strings.Contains(err.Error(), "UNIQUE constraint failed: bookmark.url") {
```

Comment:
This is only temporary, so make it work with all database engines, as that error code is specific to SQLite.

- SQLite: `UNIQUE constraint failed: x`
- MySQL/MariaDB: `Duplicate entry 'x' for key 'x'`
- PostgreSQL: `duplicate key value violates unique constraint "x"`

```suggestion
                errStr := strings.ToLower(err.Error())
		if (strings.Contains(errStr, "unique constraint failed") || strings.Contains(errStr, "duplicate entry")) && 
   strings.Contains(errStr, "duplicate key value violates unique constraint") {
```